server/src/services/server.service.ts

```diff
@@ -154,4 +159,34 @@ export class ServerService implements OnEvents {
       sidecar: Object.keys(mimeTypes.sidecar),
     };
   }
+
+  async deleteLicense(): Promise<void> {
+    await this.systemMetadataRepository.delete(SystemMetadataKey.LICENSE);
+  }
+
+  async getLicense(): Promise<LicenseKeyDto | null> {
+    return this.systemMetadataRepository.get(SystemMetadataKey.LICENSE);
+  }
+
+  async setLicense(license: LicenseKeyDto): Promise<LicenseResponseDto> {
```

Comment:
probably just rename this to dto

Reply:
Not sure what you mean here, rename what to dto?